en/hosting_guide/configuration.rst

```diff
@@ -3,6 +3,233 @@ Server Configuration
 
 To know more about server configuration see `requirements`_.
 
+Apache
+------
+Open Orchestra has just need  the default apache configuration
+
+Virtual Host
+~~~~~~~~~~~~
+
+Back Office Virtual Host:
+
+.. code-block::
+
+    <VirtualHost *:80>
+        ServerAdmin webmaster@localhost
+        ServerName admin.openorchestra.dev
+
+        DocumentRoot /var/www/openorchestra/web
+            <Directory />
+            Options FollowSymLinks
+            AllowOverride None
+        </Directory>
+        <Directory /var/www/openorchestra/web>
+            Options -Indexes +FollowSymLinks -MultiViews
+            AllowOverride All
+            Order allow,deny
+            allow from all
+        </Directory>
+
+        <IfModule mod_rewrite.c>
+            RewriteEngine On
+        </IfModule>
+
+        ErrorLog /var/log/apache2/admin.openorchestraError.log
+
+        # Possible values include: debug, info, notice, warn, error, crit,
+        # alert, emerg.
+        LogLevel warn
+
+        CustomLog /var/log/apache2/admin.openorchestraAccess.log combined
+    </VirtualHost>
+
+
+Front Office Virtual Host:
+
+.. code-block::
+
+    <VirtualHost *:80>
+        ServerAdmin webmaster@localhost
+        ServerName front.openorchestra.dev
+
+        DocumentRoot /var/www/front-openorchestra/web
+            <Directory />
+            Options FollowSymLinks
+            AllowOverride None
+        </Directory>
+        <Directory /var/www/front-openorchestra/web>
+            Options -Indexes +FollowSymLinks -MultiViews
+            AllowOverride All
+            Order allow,deny
+            allow from all
+        </Directory>
+
+        <IfModule mod_rewrite.c>
+            RewriteEngine On
+            RewriteRule ^/sitemap.xml /1/sitemap.xml [L]
+            RewriteRule ^/robots.txt /1/robots.txt [L]
+        </IfModule>
```

Comment:
You should remove that part as it depends on several things:
. The fact that sitemap.xml and robots.txt has been contributed and dumped
. The siteId

Anyway that part is described in a specific doc